app/controllers/billing_profiles_controller.rb

```diff
@@ -52,10 +52,10 @@ def update
   # DELETE /billing_profiles/aa450f1a-45e2-4f22-b2c3-f5f46b5f906b
   def destroy
     @billing_profile.destroy
-
-    respond_to do |format|
-      format.html { redirect_to billing_profiles_path, notice: t(:deleted) }
-      format.json { head :no_content }
+    if @billing_profile.destroyed?
```

Comment:
https://github.com/internetee/auction_center/commit/29678320460e2f9972edf8e6bae2402dd451c86f

Quite similar case. Consider using `destroy!`. If there is some description involved (about the reason of being unable to delete), consider moving it to UI. If something cannot be deleted, it may worth to restrict this action and give some hint to a user in UI.